docs/src/main/asciidoc/_high-availability/subsystem-support/Infinispan_Subsystem.adoc

```diff
@@ -20,14 +20,22 @@ e.g. To create a new cache container that loads marshallers from the "org.bar" m
 /subsystem=infinispan/cache-container=foo:add(module=org.foo)
 ----
 
-A cache container may designate a specific cache as its default cache, i.e. the cache returned via http://docs.jboss.org/infinispan/9.2/apidocs/org/infinispan/manager/CacheContainer.html#getCache--[CacheContainer.getCache()]:
+A cache container may designate a specific cache as its default cache, i.e. the cache returned via https://docs.jboss.org/infinispan/11.0/apidocs/org/infinispan/manager/CacheContainer.html#getCache()[CacheContainer.getCache()]:
 
 e.g. To set "bar" as the default cache of the "foo" container:
 [source,options="nowrap"]
 ----
 /subsystem=infinispan/cache-container=foo:write-attribute(name=default-cache, value=bar)
 ----
 
+A cache container can be injected directly into Jakarta EE applications using the `@Resource` annotation, e.g.
+
+[source,java,options="nowrap"]
+----
+@Resource(lookup = "java:jboss/infinispan/container/foo")
+private org.infinispan.manager.EmbeddedCacheManager container;
+----
+
```

Comment:
This should mention that any caches created from this cache manager will not have a server managed lifecycle, thus the application will have to manage the cache lifecycle manually.